includes/Core/Modules/Modules.php

```diff
@@ -233,6 +207,7 @@ function( Module $module ) {
 			}
 		);
 
+		$this->rest_modules_controller->register();
```

Comment:
This property is being declared dynamically here which is deprecated in newer versions of PHP. Please declare above, but also let's use `rest_controller` for consistency.